src/sprite_store.h

```diff
@@ -32,25 +32,20 @@ class ImageData;
  */
 class RcdBlock {
 public:
-	RcdBlock();
-	virtual ~RcdBlock();
-
-	RcdBlock *next; ///< Pointer to next block.
+	RcdBlock() = default;
+	virtual ~RcdBlock() = default;
 };
 
 typedef std::map<uint32, ImageData *> ImageMap; ///< Map of loaded image data blocks.
 
 /** Texts of objects. */
 class TextData : public RcdBlock {
 public:
-	TextData();
-	~TextData();
```

Comment:
I am wondering whether you should make all these destructors just `= default`. Might be a bit more work and not always needed, but at least it is then never missing and never forgotten.

Reply:
Nothing to gain, IMHO. Not declaring a custom ctor or dtor is the same as declaring it as `default`. Only base classes need a dtor that is explicitly marked `virtual`, for other classes it can be omitted to remove clutter.